ts/screens/authentication/IdpLoginScreen.tsx

```diff
@@ -96,32 +104,84 @@ const onNavigationStateChange = (
 class IdpLoginScreen extends React.Component<Props, State> {
   constructor(props: Props) {
     super(props);
-
     this.state = {
-      requestState: pot.noneLoading
+      requestState: pot.noneLoading,
+      startingMs: new Date().getTime()
     };
   }
 
-  private handleOnError = (): void =>
+  private getSpidId = (): string =>
```

Comment:
This is `idpId`, SPID ID is something else (a unique user ID in the Idp namespace)